README.md

```diff
@@ -40,7 +40,7 @@ contribution to the total variance of the output.
 
 To install the package using pip, execute the following command:
 
-```shell script
```

Comment:
Removing this annotation leads to a non execution with the moose in Pycharm.

Reply:
Fixed 👍